source/index.txt

```diff
@@ -8,6 +8,22 @@ The `MongoDB Connector for Spark
 <https://www.mongodb.com/products/spark-connector>`_ provides
 integration between MongoDB and Apache Spark.
 
+.. note::
+
+   Version 10.x of the MongoDB Connector for Spark is an all-new 
+   connector based on the latest Spark API. Install and migrate to 
+   version 10.x to take advantage of new capabilities, such as tighter 
+   integration with Spark 
+   :ref:`Structured Streaming <spark-structured-streaming>`.
```

Comment:
Can we move the word Spark into the ref itself? As follows:
:ref:`Spark Structured Streaming <spark-structured-streaming>`.

Reply:
Done